.circleci/config.yml

```diff
@@ -39,7 +39,7 @@ commands:
       - run:
           name: Rust Clippy
           command: |
-            cargo clippy --all --all-targets --all-features -- -D warnings
+            cargo clippy --all --all-targets --all-features -- -D warnings -A clippy::result_large_err
```

Comment:
Could you also add this to the [one](https://github.com/mozilla-services/contile/blob/main/Makefile#L20) in the Makefile?

Reply:
Looks like we need to wrap `tags` into a Box as well. I've pushed a fix on top of JR's patch. Mind taking a quick look?